vulcanize.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Script to vulcanize all the Polymer files into one big blob.
```

Comment:
The tests were passing before, so perhaps it isn't needed?? I just added it anyway.